addons/html_editor/static/tests/delete/forward.test.js

```diff
@@ -1098,7 +1098,8 @@ describe("Selection not collapsed", () => {
             stepFunction: deleteForward,
             // The flagged 200B is there to preserve the font so if we
             // write now, we still write in the font element's style.
-            contentAfterEdit: '<h1><i data-oe-zws-empty-inline="">[]\u200B</i><br></h1>',
+            contentAfterEdit:
+                '<h1 placeholder="Heading 1" class="o-we-hint"><i data-oe-zws-empty-inline="">[]\u200B</i><br></h1>',
```

Comment:
@dmo-odoo Before `isEmpty` is used to check blocks is empty and it returns false for this test and that is why placeholder does not displayed before my PR.

Reply:
But the node was _somewhat_ empty, depending on the definition we want to use... ^^' cc @FrancoisGe Is it intended that `isEmpty` returns `false` in that case? If yes, was it intended to use `isEmpty` for the placeholder hints or was it more likely an oversight due to a miscomprehension of what `isEmpty` does?

Reply:
> But the node was _somewhat_ empty, depending on the definition we want to use... ^^' cc @FrancoisGe Is it intended that `isEmpty` returns `false` in that case? If yes, was it intended to use `isEmpty` for the placeholder hints or was it more likely an oversight due to a miscomprehension of what `isEmpty` does?

I'm not sure I see the problem here ? We've got a heading that doesn't contain any text, so it's good to display the placeholder, isn't it ?

Reply:
@FrancoisGe Indeed we want the placeholder to be shown in that case. I ask @dhba-odoo how comes the placeholder were not shown in that case in `html_editor` before his PR and he told me that is was because `isEmpty` was used instead of `isEmptyBlock` and `isEmpty` returns `false` in that case whereas `isEmptyBlock` returns `true`.

I asked you two questions:
1. Was it intented that `isEmpty` returns `false` in that particular case? 
2. If the answser to 1 is "yes", then was it intended to use `isEmpty` for displaying the placeholder instead of `isEmptyBlock` or do you think it was merely an oversight from the API translation?

Reply:
@dmo-odoo To be honest, I don't have the answer. We've worked iteratively on quite a few situations. So there are probably still mistakes